lib/ddtrace/propagation/distributed_headers/b3.rb

```diff
@@ -0,0 +1,46 @@
+require 'ddtrace/ext/distributed'
+require 'ddtrace/propagation/distributed_headers/base'
+require 'ddtrace/propagation/distributed_headers/headers'
+
+module Datadog
+  module DistributedHeaders
```

Comment:
Now that distributed tracing has become more sophisticated, perhaps this namespace should reflect the category a bit better? Maybe rename this to `DistributedTracing`? Then put headers under `Datadog::DistributedTracing::Headers::B3`? What do you think?

Reply:
No strong preference

Reply:
I think then it would be a good idea to change this name.

Reply:
renamed